apps/prairielearn/src/ee/lib/ai-grading.sql

```diff
@@ -108,3 +110,24 @@ ORDER BY
   embedding <=> $embedding
 LIMIT
   $limit;
+
+-- BLOCK select_rubric_for_grading
+SELECT
+  ri.*
```

Comment:
We generally avoid wildcard selects like this, in favor of explicitly saying which columns you want.

Reply:
That is not actually true. Generally speaking, we select whole objects so that we can use the schemas from `db-types.ts` as-is. We only select individual columns when we're doing particularly complex queries, or when we want to avoid sending large amounts of data over the wire that won't be needed (which generally means params/submitted_answers/etc. for variants + submissions).

Reply:
My bad, I thought this was getting fed into a custom type defined in the file, but I was mixing it up with something else.